packages/core/src/configuration.ts

```diff
@@ -138,6 +138,9 @@ const chart: BaseChartOptions = {
 	legend,
 	style: {
 		prefix: "cc"
+	},
+	data: {
+		groupIdentifier: "group"
```

Comment:
Maybe `groupMapsTo`? `groupIdentifier` isn't bad, but if everything else is `mapsTo` and `xMapsTo` maybe we want to change it?